core/src/worker/workflow/machines/signal_external_state_machine.rs

```diff
@@ -234,7 +234,9 @@ impl WFMachinesAdapter for SignalExternalMachine {
             SignalExternalCommand::Failed(f) => {
                 let reason = match f {
                     SignalExternalWorkflowExecutionFailedCause::Unspecified => "unknown",
-                    SignalExternalWorkflowExecutionFailedCause::ExternalWorkflowExecutionNotFound => "it was not found"
+                    SignalExternalWorkflowExecutionFailedCause::ExternalWorkflowExecutionNotFound
+                    | SignalExternalWorkflowExecutionFailedCause::NamespaceNotFound =>
```

Comment:
What will happen in older SDKs if they get this variant?

Reply:
I've started creating `UNKOWN` variants for user facing enums to avoid unexpected behavior or exceptions.

Reply:
It'll be unspecified since that's the default proto value.